.github/workflows/CI.yml

```
@@ -92,3 +92,14 @@ jobs:
           MAINNET_URL: ${{ secrets.MAINNET_PROVIDER_URL }}
           DEBUG: usedapp*
         working-directory: packages/docs
+  check-mixed-changesets:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '16.x'
+      - run: npm i -g pnpm@7.1.9
+      - run: pnpm install --frozen-lockfile
```

```suggestion
      - run: pnpm install --frozen-lockfile --filter="usedapp"
```
So that it is faster